src/main/scala/org/scalameter/PerformanceTest.scala

```diff
@@ -91,7 +91,7 @@ object PerformanceTest {
    */
   trait HTMLReport extends PerformanceTest {
     import reporting._
-    def persistor: Persistor = new persistence.SerializationPersistor
+    def persistor: Persistor = new persistence.GZIPJSONSerializationPersistor
```

Comment:
The consequence of doing this is potentially breaking existing tests out there.
When we get to this point, please do this change in a separate commit - it might be necessary to introduce a new test template, or provide a migration tool from the old formats (the first is probably simpler).